assets/scss/typography.scss

```diff
@@ -1,54 +0,0 @@
-@import url(https://fonts.googleapis.com/css2?family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap);
-
-$pt-sans: 'PT Sans', sans-serif;
-$fonts: (
-        'pt-sans' : $pt-sans
-);
-
-@each $name, $font in $fonts {
-  .primary-font--#{$name} {
-    font-family: $font;
-
-    //GovUK Font Override
-    .govuk-body,
-    .govuk-details,
-    .govuk-accordion__open-all,
-    .govuk-accordion__section-heading,
-    .govuk-accordion__section-button {
-      font-family: $font;
-    }
-  }
-}
-
-//PT Sans Specific font sizes - Font size maybe something we want to add to customizer
-.primary-font--pt-sans {
```

Comment:
Happy to discuss. But I think we still need this file? Have you moved the font overrides e.g. accordion..used by the accordion block to another file? Also we may add other fonts in the future.

Reply:
I have made the font overrides very general, in the new `gds-design-system-settings.scss` file.  The overrides here weren't catching everything, so I made this general and if we need to override it later we can.  

```
#primary {
    font-family: $govuk-font-family;